.github/actions/gen-install-scripts/entrypoint.sh

```diff
@@ -14,7 +14,7 @@ mkdir -p "${crds_dir}"
 mkdir -p "${openshift}"
 
 # Generate configuration and save it to `all-in-one`
-controller-gen crd:crdVersions=v1 rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+controller-gen crd:crdVersions=v1 rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases crd:allowDangerousTypes=true
```

Comment:
I don't think this is a good idea. It is better to store floats in strings.

Reply:
No need to worry, this is a temporary thing 😅